pvacseq/lib/convert_vcf.py

```diff
@@ -105,6 +155,38 @@ def main(args_input = sys.argv[1:]):
             transcript_expns[row['tracking_id']] = row
         args.transcript_expn_file.close()
 
+    coverage = {}
+    if args.normal_snvs_coverage_file is not None:
+        if 'snvs' not in coverage:
+            coverage['snvs'] = {}
+        coverage['snvs']['normal'] = parse_bam_readcount_file(args.normal_snvs_coverage_file)
+        args.normal_snvs_coverage_file.close()
+    if args.normal_indels_coverage_file is not None:
+        if 'indels' not in coverage:
+            coverage['indels'] = {}
+        coverage['indels']['normal'] = parse_bam_readcount_file(args.normal_indels_coverage_file)
+        args.normal_indels_coverage_file.close()
+    if args.tdna_snvs_coverage_file is not None:
+        if 'snvs' not in coverage:
+            coverage['snvs'] = {}
+        coverage['snvs']['tdna'] = parse_bam_readcount_file(args.tdna_snvs_coverage_file)
+        args.tdna_snvs_coverage_file.close()
+    if args.tdna_indels_coverage_file is not None:
+        if 'indels' not in coverage:
+            coverage['indels'] = {}
+        coverage['indels']['tdna'] = parse_bam_readcount_file(args.tdna_indels_coverage_file)
+        args.tdna_indels_coverage_file.close()
+    if args.trna_snvs_coverage_file is not None:
+        if 'snvs' not in coverage:
+            coverage['snvs'] = {}
+        coverage['snvs']['trna'] = parse_bam_readcount_file(args.trna_snvs_coverage_file)
+        args.trna_snvs_coverage_file.close()
+    if args.trna_indels_coverage_file is not None:
+        if 'indels' not in coverage:
+            coverage['indels'] = {}
+        coverage['indels']['trna'] = parse_bam_readcount_file(args.trna_indels_coverage_file)
+        args.trna_indels_coverage_file.close()
```

Comment:
I pushed a commit that does something similar albeit in place and inside of some for loops so that we don't need to pass the coverage dictionary around.